modules/adqueryBidAdapter.js

```diff
@@ -0,0 +1,215 @@
+        crossOrigin: true
+      }
+    };
+  },
+
+  /**
+   * @param {*} response
+   * @param {ServerRequest} request
+   * @return {Bid[]}
+   */
+  interpretResponse: (response, request) => {
+    utils.logInfo(response);
+    utils.logInfo(request);
+    const res = response && response.body && response.body.data;
+
+    if (!res) {
+      return [];
+    }
+
+    let bidResponses = [];
+    let bidResponse = {
+      requestId: res.requestId,
+      cpm: res.cpm,
+      width: res.mediaType.width,
+      height: res.mediaType.height,
+      creativeId: res.creationId,
+      dealId: res.dealid || '',
+      currency: res.currency || ADQUERY_DEFAULT_CURRENCY,
+      netRevenue: ADQUERY_NET_REVENUE,
+      ttl: ADQUERY_TTL,
+      referrer: '',
+      ad: '<script src="' + res.adqLib + '"></script>' + res.tag,
+      mediaType: res.mediaType.name || 'banner',
+      meta: {
+        advertiserDomains: res.adDomains && res.adDomains.length ? res.adDomains : [],
+        mediaType: res.mediaType.name || 'banner',
+      }
+
+    };
+    bidResponses.push(bidResponse);
+
+    utils.logInfo(ADQUERY_LOG_PREFIX + 'bidResponses', bidResponses);
+    return bidResponses;
+  },
+
+  /**
+   * @param {TimedOutBid} timeoutData
+   */
+  onTimeout: (timeoutData) => {
+    if (timeoutData == null) {
+      return;
+    }
+    utils.logInfo(ADQUERY_LOG_PREFIX + 'onTimeout', timeoutData);
+    let params = {
+      bidder: timeoutData.bidder,
+      bId: timeoutData.bidId,
+      adUnitCode: timeoutData.adUnitCode,
+      timeout: timeoutData.timeout,
+      auctionId: timeoutData.auctionId,
+    };
+    let adqueryRequestUrl = utils.buildUrl({
+      protocol: 'https',
+      hostname: ADQUERY_BIDDER_DOMAIN,
+      pathname: '/prebid/eventTimeout',
+      search: params
+    });
+    utils.logInfo(ADQUERY_LOG_PREFIX + ': onTimeout called');
+    utils.triggerPixel(adqueryRequestUrl);
+  },
+
+  /**
+   * @param {Bid} bid
+   */
+  onBidWon: (bid) => {
+    utils.logInfo(ADQUERY_LOG_PREFIX + 'onBidWon', bid);
+    const bidString = JSON.stringify(bid);
+    const encodedBuf = window.btoa(bidString);
+
+    let params = {
+      q: encodedBuf,
+    };
+    let adqueryRequestUrl = utils.buildUrl({
+      protocol: 'https',
+      hostname: ADQUERY_BIDDER_DOMAIN,
+      pathname: '/prebid/eventBidWon',
+      search: params
+    });
+    utils.logInfo(ADQUERY_LOG_PREFIX + ' onBidWon called');
+    utils.triggerPixel(adqueryRequestUrl);
+  },
+
+  /**
+   * @param {Bid} bid
+   */
+  onSetTargeting: (bid) => {
+    utils.logInfo(ADQUERY_LOG_PREFIX + 'onSetTargeting', bid);
+
+    let params = {
+      bidder: bid.bidder,
+      width: bid.width,
+      height: bid.height,
+      bid: bid.adId,
+      mediaType: bid.mediaType,
+      cpm: bid.cpm,
+      requestId: bid.requestId,
+      adUnitCode: bid.adUnitCode,
+      adserverTargeting: getNestedParam(bid.adserverTargeting),
+    };
+
+    let adqueryRequestUrl = utils.buildUrl({
+      protocol: 'https',
```

Comment:
Same as previous comment, you have `ADQUERY_BIDDER_DOMAIN_PROTOCOL`.

Reply:
I fixed